pkg/common/common_test.go

```diff
@@ -42,6 +42,8 @@ func TestCommon(t *testing.T) {
 	Convey("test dirExists()", t, func() {
 		exists := common.DirExists("testdir")
 		So(exists, ShouldBeFalse)
+		tempDir := t.TempDir()
+		test.CopyTestFiles("../../test/data", tempDir)
 
 		file, err := os.Create("file.txt")
```

Comment:
I guess the file should be created under tempDir. By why copy the other files at line 46?